lang/en/docs/materials/overview.md

```diff
@@ -30,7 +30,7 @@ The [Data](data.md) section contains an example of JSON representation of a mate
 
 ## Classification
 
-Materials are distinguished based on whether they are [crystalline](classification/crystalline.md) or [non-crystalline](classification/non-crystalline.md).
+Materials are distinguished based on whether they are [crystalline](classification/crystalline.md), [non-crystalline](classification/non-crystalline.md), or [non-periodic](classification/non-periodic.md). An overview of classifications can be found [here](classification/overview.md)
```

Comment:
We don't really support non-crystalline materials, so let's just clean up all that related stuff, please. Should be just periodic and non-periodic structures.